pkg/scheduler/plugins/util/util.go

```diff
@@ -103,7 +105,16 @@ func GetInqueueResource(job *api.JobInfo, allocated *api.Resource) *api.Resource
 			if api.IsCountQuota(rName) || !v1helper.IsScalarResourceName(rName) {
 				continue
 			}
-
+			ignore := false
+			for _, ignoredDevice := range api.IgnoredDevicesList {
```

Comment:
What is the IgnoreDevicesList used for?

Reply:
It is a list of custom-defined resources, that vc-scheduler should ignore, instead of searching corresponding device-plugin to process it